table/queries/derived_satellite_scans.sql

```diff
@@ -1,52 +1,52 @@
 # https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-6
 CREATE TEMP FUNCTION ClassifySatelliteRCode(rcode INTEGER) AS (
   CASE
-    WHEN rcode = 1 THEN "⚠️dns/rcode:FormErr"
-    WHEN rcode = 2 THEN "⚠️dns/rcode:ServFail"
+    WHEN rcode = 1 THEN "❗️dns/rcode:FormErr"
+    WHEN rcode = 2 THEN "❓dns/rcode:ServFail"
     WHEN rcode = 3 THEN "❗️dns/rcode:NXDomain"
-    WHEN rcode = 4 THEN "⚠️dns/rcode:NotImp"
+    WHEN rcode = 4 THEN "❗️dns/rcode:NotImp"
     WHEN rcode = 5 THEN "❗️dns/rcode:Refused"
```

Comment:
Refused is unexpected nevertheless for a functioning site.

We must have a clear criteria to decide on the icons. What we landed on was to always use ❗️ for unexpected results, unless either of these are true:

1. There's an expectation of a certain level of failures, like timeouts of Servfail
2. We are not completely sure about what to expect, as in page body match

Those indicate gaps in the methodology, and merit the dubious expectation marker.
We should be documenting all those cases though.

We shouldn't use the dubious marker unless we have documented cases where the expectation is not clear.

We can discuss tweaking the criteria, but it must be clear.

Reply:
By the way, with the Servfail retry logic @ramakrishnansr added, we should consider using ❗️ for it once it lands in production, since it's clearly unexpected to have it fail multiple times in a row.

Reply:
I don't think Refused is a clear sign of failure for a working site - As mentioned in the link shared above, and based on manual investigation of the data, the appearance of the Refused error seems to be non-deterministic, and hence should warrant the dubious marker. 

The criteria can be: 
1. Deterministic failure for a certain domain where it is inaccessible - Unexpected
2. Non-deterministic failure where the failure may not be due to the domain - Dubious.